### ISSUE TITLE
add `--range` keyword argument to meds making for testing

### ISSUE THREAD

**Issue by beckermr** [beckermr/pizza-cutter#155]

**Comment by esheldon** [comment]
I can implement this.  how about
```python
@click.option(                                                                                                                                                                                                              
    '--slice-range', default=None, type=str,                                                                                                                                                                                      
    help='range of slices to process, as a python style slice with exclusive last element start:end+1')
```

**Comment by beckermr** [comment]
Let's use this so it matches the metadetect options:

```
@click.option(
    '--range', 'meds_range', default=None, type=str,
    help=(
        'the range of entries in the MEDS file to process - '
        'It is specified using normal python slicing conventions, '
        'so that `--range="0:10"` is the first 10 entries in the file, etc.'))
```

**Comment by beckermr** [comment]
The function parameter should be `meds_range` in this case.

**Comment by esheldon** [comment]
with click I think this is going to clobber the `range` function

**Comment by beckermr** [comment]
closed by #227
